pkg/detectors/sentrytoken/sentrytoken.go

```diff
@@ -19,54 +19,65 @@ type Scanner struct {
 	client *http.Client
 }
 
+type Response []Organization
+
+type Organization struct {
+	ID   string `json:"id"`
+	Name string `json:"name"`
+}
+
 // Ensure the Scanner satisfies the interface at compile time.
 var _ detectors.Detector = (*Scanner)(nil)
 
 var (
 	defaultClient = common.SaneHttpClient()
 
 	// Make sure that your group is surrounded in boundary characters such as below to reduce false positives.
-	keyPat = regexp.MustCompile(detectors.PrefixRegex([]string{"sentry"}) + `\b([a-f0-9]{64})\b`)
+	keyPat = regexp.MustCompile(`\b(sntryu_[a-f0-9]{64})\b`)
 
-	errUnauthorized = fmt.Errorf("token unauthorized")
+	forbiddenError = "You do not have permission to perform this action."
 )
 
+func (s Scanner) getClient() *http.Client {
+	if s.client != nil {
+		return s.client
+	}
+
+	return defaultClient
+}
+
 // Keywords are used for efficiently pre-filtering chunks.
 // Use identifiers in the secret preferably, or the provider name.
 func (s Scanner) Keywords() []string {
-	return []string{"sentry"}
+	return []string{"sentry", "sntryu"}
 }
 
 // FromData will find and optionally verify SentryToken secrets in a given set of bytes.
 func (s Scanner) FromData(ctx context.Context, verify bool, data []byte) (results []detectors.Result, err error) {
 	dataStr := string(data)
 
-	matches := keyPat.FindAllStringSubmatch(dataStr, -1)
+	// find all unique auth tokens
+	var uniqueAuthTokens = make(map[string]struct{})
 
-	for _, match := range matches {
-		if len(match) != 2 {
-			continue
-		}
-		resMatch := strings.TrimSpace(match[1])
+	for _, authToken := range keyPat.FindAllStringSubmatch(dataStr, -1) {
+		uniqueAuthTokens[authToken[1]] = struct{}{}
+	}
+
+	for authToken := range uniqueAuthTokens {
 		s1 := detectors.Result{
 			DetectorType: detectorspb.DetectorType_SentryToken,
-			Raw:          []byte(resMatch),
+			Raw:          []byte(authToken),
+			ExtraData:    make(map[string]string),
```

Comment:
There is only one case where `verifyToken` returns non-nil `extraData`, why not assign it directly instead of creating a blank map for every potential result?

e.g., https://github.com/trufflesecurity/trufflehog/blob/dde8f8a06d5463de2f2cce5e7a8afdcdfccf6b7b/pkg/detectors/huggingface/huggingface.go#L56